docs/guides/add-onboarding-flow.mdx

```diff
@@ -72,7 +72,6 @@ Note that the following example protects all routes except one. This is so that
 
 ```tsx filename="src/middleware.ts"
 import { clerkMiddleware, createRouteMatcher } from "@clerk/nextjs/server";
-import { redirectToSignIn } from "@clerk/nextjs/server";
```

Comment:
Thank you! Can you remove it where it's destructured on line 81?

Reply:
It can't be removed as it would result in an error at line 90

Reply:
Ah, I see I missed that. Doesn't that mean this example does use redirectToSignIn?

Reply:
Ah, I see now : 👍